src/Examples/Advanced/CustomProperties/Parameter_estimation_PR.ipynb

```diff
@@ -191,13 +190,17 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "2021-08-02 12:34:06 [INFO] idaes.init.fs.state_block: Starting initialization\n",
-      "2021-08-02 12:34:06 [INFO] idaes.init.fs.state_block: Dew and bubble point initialization: optimal - Optimal Solution Found.\n",
-      "2021-08-02 12:34:06 [INFO] idaes.init.fs.state_block: Equilibrium temperature initialization completed.\n",
-      "2021-08-02 12:34:06 [INFO] idaes.init.fs.state_block: State variable initialization completed.\n",
-      "2021-08-02 12:34:06 [INFO] idaes.init.fs.state_block: Phase equilibrium initialization: optimal - Optimal Solution Found.\n",
-      "2021-08-02 12:34:07 [INFO] idaes.init.fs.state_block: Property initialization: optimal - Optimal Solution Found.\n",
-      "2021-08-02 12:34:07 [INFO] idaes.init.fs.state_block: Property package initialization: optimal - Optimal Solution Found.\n"
+      "2022-08-23 16:38:18 [INFO] idaes.init.fs.state_block: Starting initialization\n",
+      "WARNING (W1002): Setting Var\n",
```

Comment:
This warning and subsequent warnings concerning the variable bounds will need to be addressed in the property package module, if you don't want them to print here.